### PR TITLE
Governance page

### PR DESCRIPTION
**Pull request by BobHarper1**

### CODE REVIEW

**Comment by BobHarper1** [comment]
ok @morchickit?

Two issues still to sort:

- there's some text formatting (e.g. bold) in the pdf original thats not in the markdown
- still have to find a solution to the footnotes thing

**Comment by morchickit** [comment]
All good to go from my side!

**Comment by BobHarper1** [comment]
Re internal issue #12696

File: documentation/governance.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
+# Governance and Revision Control 
+```eval_rst
+
+.. admonition:: Download
+
+  You can also download this document in PDF format :download:`here <../assets/governance_and_review_process.pdf>`.
```
Comment (Bjwebb) [inline documentation/governance.md]: Not sure this is very good from an SSOT perspective.

Comment (BobHarper1) [reply documentation/governance.md]: @morchickit thoughts? Is it necessary to have a PDF document for download?

Comment (morchickit) [reply documentation/governance.md]: I see Ben's point. We can remove the PDF then.

Comment (BobHarper1) [reply documentation/governance.md]: Removed now.

**Pull request merged**